core/src/main/scala/filodb.core/store/ChunkSink.scala

```diff
@@ -64,8 +64,8 @@ class ChunkSinkStats {
   private val chunkBytesHist     = Kamon.histogram("chunk-bytes-per-call")
   private val chunkLenHist       = Kamon.histogram("chunk-length")
 
-  private val numIndexWriteCalls = Kamon.counter("index-write-calls-num")
-  private val indexBytesHist     = Kamon.histogram("index-bytes-per-call")
+  private val numIngestionWriteCalls = Kamon.counter("ingestion-write-calls-num")
```

Comment:
I also think these should still be `index` and not `ingestion`....  `ingestion-write-calls` is gonna be really confusing